content/tutorials/token-authentication.textile

```diff
@@ -685,6 +821,19 @@ blang[php].
 
   And then run the demo locally by adding your "Ably API key":https://support.ably.io/support/solutions/articles/3000030502-setting-up-and-managing-api-keys to @ably.php@ and running the demo @php -S 0.0.0.0:8000@ inside @public@ directory.
 
+blang[python].
+  The complete source code for each step of "this tutorial is available on Github":https://github.com/ably/tutorials/commits/token-authentication-python.
+
+  We recommend that you clone the repo locally:
+
+  bc[sh]. git clone https://github.com/ably/tutorials.git
+
+  Checkout the tutorial branch:
+
+  bc[sh]. git checkout token-authentication-python
+
+  And then run the demo locally by adding your "Ably API key":https://support.ably.io/support/solutions/articles/3000030502-setting-up-and-managing-api-keys to @server.py@ and running the demo @python server.py@. Make sure you've installed @web.py@ (@pip install web.py@)
```

Comment:
Should we not mention both dependencies i.e. ably and web.py, perhaps "Make sure you've installed the dependencies (@pip install ably web.py@)